CryptoTechReminderSystem.Test/HarvestGatewayTests.cs

```diff
@@ -74,9 +88,44 @@ public void CanGetDeveloperWithAuthentication()
                 "tingkywinky@email.com"
             );
             
-            var response = _harvestGateway.Retrieve();
+            var response = _harvestGateway.RetrieveDevelopers();
 
             _fluentSimulator.ReceivedRequests.First().Headers["Authorization"].Should().Be("Bearer " + Token);
         }
+        
+        [Test]
+        public void CanGetATimeSheet()
+        {
+            SetUpUsersTimeSheetApiEndpoint("../../../HarvestTimeEntriesApiEndpoint.json");
```

Comment:
I think you can place everything in `SetUpUsersTimeSheetApiEndpoint` in a default `SetUp` method so you don't have to call it in every unit test.